### DISCUSSION TITLE
Trivy k8s scans with trivy-server

### DISCUSSION THREAD

**Discussion by user-8f93f49a** [aquasecurity/trivy#4414] [Q&A] [answered]
### Question

Hello everyone,

We have a trivy server deployed to our k8s cluster using the [helm chart ](https://github.com/aquasecurity/trivy/tree/v0.40.0/helm/trivy) and trying to use it to scan kubernetes clusters.
I could not found a proper option in the trivy cli when calling `trivy k8s` or `trivy kubernetes` there is no option to define a trivy server url. 

Is there a way to use trivy k8s scan with a trivy-server?
Like a `trivy k8s --server <remote-trivy-server-url:4954>`.

Thanks,
Best regards,
Dmitriy

### Target

Kubernetes

### Scanner

Vulnerability

### Output Format

JSON

### Mode

Client/Server

### Operating System

linux

### Version

```bash
Version: 0.37.3
Vulnerability DB:
  Version: 2
  UpdatedAt: 2023-05-11 06:07:21.356132135 +0000 UTC
  NextUpdate: 2023-05-11 12:07:21.356131735 +0000 UTC
  DownloadedAt: 2023-05-11 10:19:32.708735 +0000 UTC
```

**Comment by knqyf263** [comment, marked as answer]
We'd recommend [Trivy Operator](https://github.com/aquasecurity/trivy-operator) for the purpose.

**Reply by user-8f93f49a** [reply]
Thanks for your response. Got it. Will think about trivy-operator
